sql/moz-fx-data-shared-prod/telemetry_derived/main_nightly_v1/query.sql

```diff
@@ -11,4 +11,6 @@ FROM
   telemetry_stable.main_v5
 WHERE
   normalized_channel = 'nightly'
-  AND DATE(submission_timestamp) = @submission_date
+  {% if not is_init() %}
+    AND DATE(submission_timestamp) = @submission_date
+  {% endif %}
```

Comment:
I think is_init would fail due to not having a partition filter
```suggestion
  {% if not is_init() %}
    AND DATE(submission_timestamp) = @submission_date
  {% else %}
    AND FALSE
  {% endif %}
```